spec/services/decisions/case_decision_tree_spec.rb

```diff
@@ -317,21 +322,37 @@
 
         it { is_expected.to have_destination('/steps/income/employment_status', :edit, id: crime_application) }
       end
-    end
 
-    context 'and the application requires evidence upload' do
-      let(:means_passported) { false }
-      let(:evidence_required) { true }
-      let(:has_benefit_evidence) { true }
+      context 'and the application requires evidence upload' do
+        let(:means_passported) { false }
+        let(:evidence_required) { true }
+        let(:has_benefit_evidence) { true }
```

Comment:
Could you add:
let(:benefit_type) { BenefitType::UNIVERSAL_CREDIT.to_s }

Just to make it clear that this is an applicant claiming they have ppt benefit but haven't passed the BC